qbft/decided.go

```diff
@@ -28,18 +28,20 @@ func (c *Controller) UponDecided(msg *SignedMessage) (*SignedMessage, error) {
 	}
 
 	// did previously decide?
-	inst := c.InstanceForHeight(msg.Message.Height)
-	prevDecided := inst != nil && inst.State.Decided
+	inst, err := c.InstanceForHeight(msg.Message.Height)
+	prevDecided := err == nil && inst != nil && inst.State.Decided
```

Comment:
error needs to be checked independently

Reply:
do we want to failed processing msg cause of this error?

Reply:
yes we do, if we can't fetch it